index.html

```diff
@@ -167,6 +168,12 @@ <h3>Recommendation Track specification repositories</h3>
         <li><a href="https://github.com/w3c/ttml2/pulse">TTML2</a></li>
         <li><a href="https://github.com/w3c/ttml3/pulse">TTML3</a></li>
         <li><a href="https://github.com/w3c/webvtt/pulse">WebVTT</a></li>
+        <li><a href="https://github.com/w3c/adpt/pulse">ADPT</a></li>
+        <li><a href="https://github.com/w3c/tt-module-cjk-ext-1/pulse">TTML Extension Module for CJK</a></li>
+        <li><a href="https://github.com/w3c/tt-module-css/pulse">TTML CSS Extensions Module</a> <span class="small">(not currently actively being worked on)</span></li>
```

Comment:
For this and the following entries, please change "Extensions Module" to "Extension Module". Here "Extension" is already understood to be one or more new underlying capabilities, plus we already have a formal definition [extension module] which is used in the singular but may refer to a collection of capabilities.

Reply:
Done.